gitium/gitium-webhook.php

```diff
@@ -17,8 +17,12 @@
 
 header( 'Content-Type: text/html' );
 define( 'SHORTINIT', true );
-//$wordpress_loader = $_SERVER['DOCUMENT_ROOT'] . '/wp-load.php';
-$wordpress_loader = filter_input(INPUT_SERVER, 'DOCUMENT_ROOT', FILTER_SANITIZE_STRING) . '/wp-load.php';
+
+$current_dir = __DIR__;
```

Comment:
We should not drop the previous approach so that we don't break backward compatibility. Also checking for `wp-content/plugins` is to narrow scoped, since the plugin can be installed in `mu-plugins` as well, and also `wp-content` can be set in `wp-config.php`.

I think we should check like this:
```
$wordpress_loader = "$current_dir/../../../wp-load.php";
if ( file_exists($wordpress_loader) ) ...
```